### PR TITLE
add Bookcopies for existing book.

### PR DESCRIPTION
**Pull request by Jopvdhoef**
Book copies can now be added for existing books in the catalog. The book components now each have a button to add these copies. 

The Trainer is prompted to add the number of copies they want to add. The default value of a new copy is "Nieuw" but this can be changed to 1 of 5 choices. After the Trainer clicks on "Opslaan", they should get a confirmation popup. After confirming, a modal with the added copies and their WTId's should be shown. 

After adding copies, the number of copies for the specified book is automatically changed.

### CODE REVIEW

File: src/app/book/book.component.html [hunk 1]
```diff
@@ -1,33 +1,23 @@
...
+    </button>
+    <h6
+      class="card-subtitle mb-2 text-muted"
+      *ngFor="let item of book?.authors"
+    >
+      {{ item }}
+    </h6>
+    <p class="card-text">{{ book?.description }}</p>
+      <app-add-copies [bookAddCopy]="book"></app-add-copies>
     </div>
```
Comment (Rik131) [inline src/app/book/book.component.html]: De knop om exemplaren toe te voegen is nu visible voor trainees

File: src/dto/ReadBookCopyDto.ts
Comment (Ilias-Github) [inline src/dto/ReadBookCopyDto.ts:1]: Deze comments kunnen weg als ze niet gebruikt worden

File: src/app/book-form/book-form.component.ts [hunk 2]
```diff
@@ -76,6 +76,7 @@ export class BookFormComponent {
         // Dit wordt uitgevoerd nadat we een reponse hebben ontvangen
         if (response.success) {
           this.bookCopyList = response.data;
+          console.log("bookCopyList", this.bookCopyList)
```
Comment (Ilias-Github) [inline src/app/book-form/book-form.component.ts]: Deze console log mag weg

File: src/app/add-copies/add-copies.component.ts [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+  onSubmit(): void {
+    let dto = {
+      states:  this.states.value,
+      bookId: this.bookAddCopy?.id
+    }
+
+    if(confirm("Weet je zeker dat je " + this.states.length + " exemplaren van het boek: '" + this.bookAddCopy?.title + "' wilt toevoegen?")) {
+      this.bookCopyService.createBookCopy(dto).subscribe(response => {
+        console.log("request", dto);
```
Comment (Ilias-Github) [inline src/app/add-copies/add-copies.component.ts:56]: Vergeet deze niet

File: src/app/book/book.component.html (hunk 1 above)
Comment (Jopvdhoef) [reply src/app/book/book.component.html]: aangepast

**Pull request merged**